### PR TITLE
Update administrative-templates-import-custom.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3884

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3896

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
@msbemba Both steps are correct. You can add both steps, like: Select **Devices** > **Configuration profiles** > **Import ADMX** > **Import**. Or, select **Devices** > **Windows** > **Configuration profiles** > **Import ADMX** > **Import**:

**Comment by msbemba** [comment]
@MandiOhlinger  Thank you. I have added both steps to the document now.

File: memdocs/intune/configuration/administrative-templates-import-custom.md [hunk 1]
```diff
@@ -105,9 +105,11 @@ Download the ADMX templates you want to import. Save these files to an easily ac
 ## Add the ADMX and ADML files
 
 1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
-2. Select **Devices** > **Configuration profiles** > **Import ADMX** > **Import**:
+2. Select **Devices** > **Configuration profiles** > **Import ADMX** > **Import** :
 
     :::image type="content" source="./media/administrative-templates-import-custom/import-admx.png" alt-text="Screenshot that shows how to add or import custom ADMX and ADML. Go to Devices > Configuration profiles > Import ADMX in Microsoft Intune and Intune admin center.":::
+    
+    Alternatively, you can also import from **Devices** > **Windows** > **Configuration profiles** > **Import ADMX**
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/administrative-templates-import-custom.md:112]:
```suggestion
    Alternatively, you can also import from **Devices** > **Windows** > **Configuration profiles** > **Import ADMX**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by MandiOhlinger — APPROVED** [review]

**Pull request merged**